servers/src/main/java/tachyon/worker/block/BlockLockManager.java

```diff
@@ -120,9 +118,9 @@ public void unlockBlock(long lockId) throws NotFoundException {
     Lock lock;
     synchronized (mSharedMapsLock) {
       LockRecord record = mLockIdToRecordMap.get(lockId);
-      if (null == record) {
-        throw new NotFoundException("Failed to unlockBlock: lockId " + lockId
-            + " has no lock record");
+      if (record == null) {
```

Comment:
It's just a convention that many people (myself included) find the if (variable == constant) form to be more readable, but my opinion is not strong not this, and it's a more or less a flavor issue.